com.github.GradienceTeam.Gradience.json

```diff
@@ -40,76 +46,48 @@
                 }
             ]
         },
-        "pypi-dependencies.json",
         {
-            "name": "blueprint-compiler",
-            "buildsystem": "meson",
-            "sources": [
+            "name" : "blueprint-compiler",
+            "buildsystem" : "meson",
+            "sources" : [
                 {
-                    "type": "git",
-                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
-                    "tag": "v0.2.0"
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                    "tag" : "v0.4.0"
                 }
             ]
         },
         {
-            "name": "libadwaita",
-            "buildsystem": "meson",
-            "config-opts": [
-                "-Dvapi=false",
-                "-Dtests=false",
-                "-Dexamples=false",
-                "-Dintrospection=enabled"
+            "name": "libsass",
+            "buildsystem": "simple",
+            "build-commands": [
+                "autoreconf --force --install",
```

Comment:
this should still be using the `autotools` build type like it was.

Reply:
Can I use the same formula as before, or should I change something in it?
By formula I mean this part:
```json
{
    "type": "script",
    "dest-filename": "autogen.sh",
    "commands": [
        "autoreconf -si"
    ]
}
```

Reply:
why did you change it?

Reply:
then upstream should be fixed instead